app/javascript/packages/analytics/index.ts

```diff
@@ -15,26 +15,24 @@ interface NewRelicGlobals {
  *
  * @return Promise resolving once event has been logged.
  */
-export async function trackEvent(event: string, payload: object = {}): Promise<void> {
+export async function trackEvent(event: string, payload?: object): Promise<void> {
```

Comment:
For context, argument defaulting transpilation is a bit verbose. I expect this would correct itself once browser support for optional arguments is good enough to avoid the syntax downgrading.

Transpiled result:

```ts
async function(e) {
  let n = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : {};
  // ...
}
```

Reply:
My first thought was, why are we transpiling this? but wow default parameters only have [93% availability on canicuse](https://caniuse.com/mdn-javascript_functions_default_parameters), that is surprising

Reply:
That is surprising!

There are ways we could optimize this a bit by [telling Babel certain "assumptions"](https://babeljs.io/docs/en/assumptions#ignorefunctionlength), but I'm not overly stressed about it broadly speaking. Hopefully with having dropped IE support and targeting browser support percentage, it won't be too long before we can use the native syntax.